### DISCUSSION TITLE
No logging data avalible.

### DISCUSSION THREAD

**Discussion by rasmuslinden** [requarks/wiki#6286] [Help / Questions]
Hello All.
Im trying to debug a wiki.js installation running on debian using postgres.
Everything seems to be working fine, however no logs are beeing generated.
I have turned on ldap debugging, but still no logs are avalible anyplace on the systems.

Can one of you point me in the right direction ?
All the best
Ras

**Comment by fnafox** [comment]
Logging depends on your configuration. 

- If you've created a systemd service for the wiki, you can see logs in systemd-journald using `journalctl -xeu wiki.service` where wiki is the name of your wiki service.
- If you're running a docker installation then `docker logs wiki` will print the logs from your container. As before wiki should be the name of your wiki.js container.
